deposit-app/src/test/java/edu/unc/lib/boxc/deposit/validate/ValidateContentModelJobTest.java

```diff
@@ -564,4 +565,43 @@ public void invalidChildAclsTest() {
             job.run();
         });
     }
+
+    @Test
+    public void workValidAspaceRefIdTest() {
+        PID objPid = makePid(CONTENT_BASE);
+        Bag objBag = model.createBag(objPid.getRepositoryPath());
+        objBag.addProperty(RDF.type, Cdr.Work);
+        objBag.addProperty(CdrAspace.refId, "2817ec3c77e5ea9846d5c070d58d402b");
+
+        PID childPid = makePid(CONTENT_BASE);
+        Resource childResource = model.getResource(childPid.getRepositoryPath());
+        childResource.addProperty(RDF.type, Cdr.FileObject);
+        Resource origResource = DepositModelHelpers.addDatastream(childResource);
+        origResource.addLiteral(CdrDeposit.stagingLocation, "path");
+        objBag.add(childResource);
+
+        depBag.add(objBag);
+
+        job.closeModel();
+
+        job.run();
+    }
+
+    @Test
+    public void fileInvalidAspaceRefIdTest() {
+        Assertions.assertThrows(InvalidAssignmentException.class, () -> {
+            doThrow(new InvalidAssignmentException()).when(aclValidator).validate(any(Resource.class));
```

Comment:
Since this test doesn't involve access control, we shouldn't need this. Ideally the job should throw an error without needing to mock anything in this case. Judging by the other tests it should be JobFailedException